app/views/chapters/_chapter_form.html.erb

```diff
@@ -53,13 +53,13 @@
   <p class="rtf-html-instructions note">
     <span class="html-notes"><%= allowed_html_instructions %></span>
     <span class="hidden rtf-notes"><%= ts('Type or paste formatted text.') %><%= link_to_help "rte-help" %></span>
-  </p>  
+  </p>
   <div class="rtf-html-field">
     <%= f.text_area :content, :id => "content", :class => "mce-editor observe_textlength large" %>
-    <%= live_validation_for_field('content', 
-          :maximum_length => ArchiveConfig.CONTENT_MAX_DISPLAYED, 
-          :minimum_length => ArchiveConfig.CONTENT_MIN, 
-          :tooLongMessage => ts('We salute your ambition! But sadly the content must be less than %{max} characters long. (Maybe you want to create a multi-chaptered work?)', :max => ArchiveConfig.CONTENT_MAX_DISPLAYED.to_s),
+    <%= live_validation_for_field('content',
+          :maximum_length => ArchiveConfig.CONTENT_MAX_DISPLAYED,
+          :minimum_length => ArchiveConfig.CONTENT_MIN,
+          :tooLongMessage => ts('We salute your ambition! But sadly each chapter of a work must be less than %{max} characters long.', :max => ArchiveConfig.CONTENT_MAX_DISPLAYED.to_s),
```

Comment:
i wanna do this :D

thank you for the pointers! i'll work through these and push the changes ✌️